setools/perm_map

```diff
@@ -146,13 +146,18 @@ class dir 26
            relabelto         w        10
              mounton         b         1
              quotaon         b         1
-              swapon         b         1
+              swapon         b        10
```

Comment:
2 thoughts here.

1. Why would `swapon` be different from `mounton`?
2. `swapon` isn't checked, so arguably it should be `n 1`

Reply:
Added commit setting all removed permissions to `n 1`